lib/laerekontrakt-generate-document-restricted-address.js

```diff
@@ -12,18 +12,10 @@ module.exports = async data => {
   const schoolInfo = getSchoolInfo({ organizationNumber: data.schoolOrganizationNumber.replace(/\D/g, '') })[0]
   const documentTemplate = getDocumentTemplate({ domain: 'minelev', templateId: 'hemmelig-adresse-kontrakt-signert' })
   const template = documentTemplate.archive
-  const docType = data.documentTemplate === 'ikke-samtale' ? 'Brev' : 'Varsel'
   const title = `Informasjonsbrev - godkjent lærekontrakt – ${date} - må sendes manuelt`
   const unofficialTitle = title
-  logger('info', ['laerekontrakt-generate-document-restricted-address', 'documentTemplate', data.documentTemplate, 'docType', docType])
-  const documentData = {
-    data: {
-      dato: date,
-      navnElev: data.studentName
-    },
-    template: documentTemplate.filePath
-  }
-  const fileData = await generateDocumentData(documentData)
+  logger('info', ['laerekontrakt-generate-document-restricted-address', 'documentTemplate', data.documentTemplate, 'title', title])
+  const fileData = await generateDocumentData(data)
```

Comment:
Her må også data som sendes inn justeres for at malen skal komme riktig ut.

Reply:
Fisket